src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -66,10 +66,11 @@ final class PhpStormStubsSourceStubber implements SourceStubber
     /** @var array<string, Node\Const_|Node\Expr\FuncCall> */
     private $constantNodes = [];
 
-    public function __construct(Parser $phpParser)
+    public function __construct(Parser $phpParser, ?string $stubsDirectory = null)
```

Comment:
I would suggest adding a named constructor, instead of changing this constructor. Test additions required.

Reply:
Named constructor added.  Let me know if the test additions are sufficient.